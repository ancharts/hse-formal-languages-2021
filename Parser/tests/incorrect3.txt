START = Lang
Lang = B + ('b' | ('b' + 'a'))*
A = 'a'*
% Test comment
