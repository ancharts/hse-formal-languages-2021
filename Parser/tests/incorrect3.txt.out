Error: It is undefined ID - B
